Library/Homebrew/formula.rb

```diff
@@ -1008,7 +1008,7 @@ def caveats
   # rarely, you don't want your library symlinked into the main prefix
   # see gettext.rb for an example
   def keg_only?
-    keg_only_reason && keg_only_reason.valid?
+    !!keg_only_reason && keg_only_reason.valid?
```

Comment:
My not supplying a reason I think would mean I want it to be `keg_only` unconditionally, just as `fails_with :clang` means it fails with clang, regardless of whether I happen to elaborate in a `do` block with a `build`, or `cause`, or both.